src/attr.rs

```diff
@@ -27,6 +30,7 @@ pub(crate) fn valid(src: &str) -> usize {
 ///
 /// Each value is paired together with an [`AttributeKind`] in order to form an element.
 #[derive(Clone, Debug, Eq, PartialEq, Default)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize), serde(transparent))]
```

Comment:
I added `serde(transparent)` to `AttributeValue` to make the serialization a bit nicer (`"foo"` instead of `{"raw": "foo"}`).